optuna/integration/pytorch_lightning.py

```diff
@@ -96,7 +94,7 @@ def on_validation_end(self, trainer: Trainer, pl_module: LightningModule) -> Non
         if trainer.is_global_zero:
             self._trial.report(current_score.item(), step=epoch)
             should_stop = self._trial.should_prune()
-        should_stop = trainer.training_type_plugin.broadcast(should_stop)
```

Comment:
I think we should update the version check at L64 from `1.5.0` to `1.6.0` since `pytorch-lightning==1.5.0` does not have `Trainer.strategy`.

| `pytorch-lightning` version | `optuna-examples/pytorch/pytorch_lightning_simple.py`  |
| --- | --- |
| 1.5.0 | NG |
| 1.6.0 | OK |
| 1.7.0 | OK |
| 1.8.0 | OK |

The error message during the `pytorch-lightning==1.5.0` was as follows:

> AttributeError: 'Trainer' object has no attribute 'strategy'

Reply:
Thank you for your comment. I updated the version constraint.